gdsfactory/cell.py

```diff
@@ -146,6 +123,11 @@ def mzi_with_bend():
         mzi_with_bend_decorated = gf.cell(mzi_with_bend)
 
     """
+    if default_decorator is not None:
+        warnings.warn(
+            "default_decorator is deprecated and will be removed soon. Use post_process instead.",
+            stacklevel=2,
```

Comment:
I think we should also use the warning category so one can filtre out deprecation warnings easily if desired
```suggestion
            "default_decorator is deprecated and will be removed soon. Use post_process instead.",
            DeprecationWarning,
            stacklevel=2,
```